enclave_manager/avalon_enclave_manager/kme/kme_enclave_manager.py

```diff
@@ -260,6 +264,9 @@ def main(args=None):
         config["KMEListener"]["bind"] = options.bind
     if options.worker_id:
         config["WorkerConfig"]["worker_id"] = options.worker_id
+    if options.cleanup_work_orders:
```

Comment:
What is this cleanup work orders parameter in config file? can you please add a comment here.